src/views/canvas/CanvasSelection.js

```diff
@@ -80,6 +77,17 @@ extend(CanvasSelection.prototype, {
     })();
   },
 
+  _drawVerticalLine: function (xStart, yStart, yEnd) {
```

Comment:
nit: Let's inline `_drawVerticalLine` within `_drawVerticalBorder`.
It's a bit hard to distinguish between the two, and I don't see `_drawVerticalLine` being used multiple times.